### ISSUE TITLE
Custom timer for Ventures

### ISSUE THREAD

**Issue by mmaine** [nvll/xivtools#38]
Ventures currently have a set timer of 60 minute refresh, but there are ventures that last less that time if you're higher level.

Ventures from 1-60 are 40 minutes and 61-70 are 50 minutes. This can be up to hours saved, depending how often it's used.

ie. Sometimes you farm ventures at a different timing, and perhaps a more robust system to allow timers set?

Example of my current ventures I do when manual:
https://i.imgur.com/SCyYCTJ.png

tl;dr: maybe allow set a venture global timer of 40, 50 or 60, or perhaps each retainer with its own customer refresh timer?

**Comment by nvll** [comment]
Ventures already supports this, but it's possible the UI for it isn't very clear.
`OPTIONS:
    -1 <r1-period>                     How many minutes a retainer's ventures take to complete (default:60)
    -0 <r10-period>
    -2 <r2-period>
    -3 <r3-period>
    -4 <r4-period>
    -5 <r5-period>
    -6 <r6-period>
    -7 <r7-period>
    -8 <r8-period>
    -9 <r9-period>
`

What this means is that if you had 3 retainers (1, 2, and 3 in the list) and their venture lenghts were 40, 50, and 60 you would use these flags:
`ventures.exe -r 1-3 -1 40 -2 50 -3 60`

It will then wake up and service each venture as they become available. Like you said, they can fail pretty far out of sync so it makes quite a difference for the 40 minute ventures.

**Comment by nvll** [comment]
Another thing worth mentioning is the bot is designed around the idea that you sent all the retainers out at the same time, but that their venture length can differ. I thought about supporting different start times, but the interface for users to deal with it was awkward and in the end I added the general `-t` option to support "I sent retainers out -t minutes ago" to handle that situation.

**Comment by mmaine** [comment]
Ah right, I just couldn't find any documentation on this.

I knew of -t, but i wasn't aware that you could set periods. good to know!
